README.md

```diff
@@ -0,0 +1,72 @@
+# Mainframe Browser
+
+A browser for the new decentralized web. Makes building and running distributed, unstoppable apps easier. Built with electron.
+
+## Overview
+
+The main components (packages) of the Mainframe Browser are as follows:
+
+- `daemon`: a background daemon process
+- `launcher`: a front-end browser (powered by electron)
+
+In addition, there are several auxiliary packages:
+
+- `app-permissions`: a grouping of permissions that can be granted to dapps
```

Comment:
Thanks for the clarification. However, can you explain the overloading of the term "dapp"? In all the conversations I've had so far, this is the word I've heard that describes what we're enabling developers to build within our launcher.

Reply:
"dApp" is the term used by Ethereum, same as "web3", from my understanding with the expectations that it can interact with the Ethereum blockchain.
Our platform will have different constraints and possibilities so I'd rather not create confusion by using a term used with other expectations.

Reply:
I think we should discuss this in a larger group, because I hear the word being tossed around in many contexts.